examples/window-app/common/src/WindowApp.cpp

```diff
@@ -469,7 +481,7 @@ void WindowApp::Cover::GotoLift(EventId action)
 
     if (EventId::LiftUp == mLiftAction)
     {
-        if (current < target)
+        if (current.Value() < target.Value())
```

Comment:
What if one of them is null?  Then this will crash.

Reply:
you right forgot those